cypress-io/cypress
# fix: Truncate spec directories and file names instead of wrapping

- Closes #21895

### User facing changelog


Spec directories and file names are now truncated if they exceed a certain width

### Additional details


In the spec list, directory paths and file names were wrapping to show the entire value rather than truncating when they overflowed max width. This lead to long file/dir names clogging up the UI and overflowing the set height for the row.

This change affects the spec lists for e2e and component testing. See the related issue for more details

### Steps to test


- Open the spec list for an application (e2e or ct) and create a directory and/or spec file with a long name (shrink window width if necessary)
- Verify that when the file/dir name is longer than the width of the spec column in the table, the name is truncated
- Verify that you can view the full name of the file/dir by hovering over the truncated text

### How has the user experience changed?


**Before**
![Screen Shot 2022-06-09 at 12 08 29 PM](https://user-images.githubusercontent.com/14275198/172900052-d2f5c440-1f21-4ec0-8df1-c07cc96df8ff.png)

**After**
![Screen Shot 2022-06-09 at 12 14 37 PM](https://user-images.githubusercontent.com/14275198/172900116-cf0f21cc-53e0-4a5d-a56f-a990c8fe59bf.png)


### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
